cinema_service/settings.py

```diff
@@ -127,14 +132,13 @@
 
 USE_TZ = False
 
-
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
The `MEDIA_ROOT` is set to a static path. Consider using `os.path.join(BASE_DIR, 'media')` to dynamically set the media root based on the project's base directory.